clis/teliod/src/main.rs

```diff
@@ -96,13 +105,21 @@ async fn main() -> Result<(), TeliodError> {
             if DaemonSocket::get_ipc_socket_path()?.exists() {
                 Err(TeliodError::DaemonIsRunning)
             } else {
+                #[cfg(feature = "qnap")]
+                let _pid_guard = qnap::PidFile::new().await?;
```

Comment:
I wonder do we have a need for pidfiles?

Reply:
That's the pattern used for QNAP applications, for start/stop routines (eg: [qnap/shared/NordSecMeshnet.sh](https://github.com/NordSecurity/libtelio/blob/main/qnap/shared/NordSecMeshnet.sh)).

good point, initially we were killing the process from the cgi script, which was in the meantime removed and we don't need the Pidfile now.. so will remove Pidfile from the app side.